### PR TITLE
Initial wording for shape rendering

### PR DESCRIPTION
**Pull request by mbasaglia**
Adds wording for shape algorithms. I used BCP 14 words to specify compliance requirements.

### CODE REVIEW

**Comment by mbasaglia** [comment]
This SHOULD match lottie-web output.

File: docs/index.md [hunk 1]
```diff
@@ -22,16 +22,34 @@ Once the draft is complete, there will be an announcement by the
...
+
+A machine-readable specification of the JSON structure is available
+as [JSON Schema](./specs/schema.md).
+
+
 ## Where to start
 
-Lottie files use JSON to structure its data so basic JSON knowledge
-is required to understand the specidification.
+Since Lottie uses JSON, basic JSON knowledge is required to understand the specidification.
```
Comment (b-wils) [inline docs/index.md:48]: Typo: speci**d**ification

**Review by b-wils — APPROVED** [review]

**Review by fmalita — CHANGES_REQUESTED** [review]
Thanks Mattia, this is awesome!

File: docs/specs/shapes.md [hunk 2]
```diff
@@ -170,3 +290,41 @@ The `ty` property defines the specific element type based on the following value
...
+
+To render trim path, implementations MUST consider the actual length of
+each shape (they MAY use approximations). Once the shapes are collected,
+the segment to render is given by the percentages $start$ and $end$.
+
+If $start$ and $end$ are equal, implementations MUST NOT render any shapes.
+
+If they are equal (implementations MAY consider very similar values as equal),
+the input shape MUST be rendered as-is.
+
```
[the remark this answers is not in the document]
Comment (mbasaglia) [reply docs/specs/shapes.md]: I reworded trim path rendering, hopefully it makes more sense now

**Comment by mbasaglia** [comment]
As a sanity check, I've written this script that given rendering instructions as per the specs, renders a lottie path shape: https://gist.github.com/mbasaglia/5e82cbbe64bbbd3ce96aadb0f952b1e9

Perhaps it would be useful to add it to the github actions to verify the instructions make sense and such but it would need more work for that.

**Comment by mbasaglia** [comment]
Shapes as rendered from the instructions:
rect
ellipse

**Review by fmalita — APPROVED** [review]

**Review by kudanai — APPROVED** [review]

**Pull request merged**